src/transport_performance/gtfs/gtfs_utils.py

```diff
@@ -60,3 +63,88 @@ def bbox_filter_gtfs(
     print(f"Filtered feed written to {out_pth}.")
 
     return None
+
+
+# NOTE: Possibly move to a more generalised utils file
+def convert_pandas_to_plotly(
+    df: pd.DataFrame, return_html=False, scheme: str = "dsc"
```

Comment:
Currently, there are no alternative schemes, providing this parameter potentially misleading. Suggest removing parameter and associated logic and keeping the hardcoded `schemes` variable.

Reply:
`return_html=False` needs type setting